src/integrationTest/groovy/com/streamr/client/StreamEndpointsSpec.groovy

```diff
@@ -208,25 +198,4 @@ class StreamEndpointsSpec extends StreamrIntegrationSpecification {
         isValid1
         !isValid2
     }
-
```

Comment:
We could  keep these two tests and convert them to use private key authentication (`createClientWithPrivateKey`())